packages/babel-parser/src/parser/statement.js

```diff
@@ -2115,6 +2115,7 @@ export default class StatementParser extends ExpressionParser {
     if (!attributes.has("type")) {
       throw this.raise(this.state.start, Errors.ModuleAttributesWithoutType);
```

Comment:
```suggestion
      this.raise(this.state.start, Errors.ModuleAttributesWithoutType);
```

Reply:
Oh actually I think that we can remove this now, since `import "foo" with x: "y";` will already throw because `x` is not a valid attribute, and `import "foo" with;` isn't valid anyway.
Even without this error, the only thing that doesn't throw is `import "foo" with type: "xx"`.

Reply:
Understood. I have now removed this condition.